src/App.test.js

```diff
@@ -0,0 +1,6 @@
+import { render } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+});
```

Comment:
- [ ] Kindly make your test is passing because at this moment this `app.test.js` is no passing. Please work on it order to meet all project requirements. :wink: 

  - ### screenshot
   ![image](https://user-images.githubusercontent.com/87197412/202477671-783a52e7-7062-499a-8ee4-2151a2b88df5.png)

Reply:
That particular file was supposed to be deleted. I will take it out of the project files